main_es.tex

```diff
@@ -12,11 +12,11 @@
 }
```

Comment:
Just to be clear: "Version" in Spanish is also "Version"?

Reply:
Corrected